src/indicator/renderer/relativeStrengthIndex.js

```diff
@@ -19,9 +19,17 @@ export default function() {
 
     var rsi = function(selection) {
 
-        multiSeries.xScale(xScale)
+        multi
+            .xScale(xScale)
             .yScale(yScale)
-            .series([rsiLine, annotations])
+            .series([annotations, rsiLine])
+            .decorate(function(g, data, index) {
```

Comment:
No was just bringing into same sequence as the stochastic just for consistency.

Force indicator added after this one is the same was as RSI was so happy to revert, was only going to make them all look as similar as possible